mon-pix/translations/en.json

```diff
@@ -1278,6 +1278,12 @@
           "img": "pôle emploi connect"
         }
       },
+      "fwb": {
+        "title": "Sign-in with FWB",
```

Comment:
Remarque : le ticket dit Log in via the FWB mais pour Pole Emploi on a Sign-in with Pole Emploi. A voir ce qu'on choisit.

Reply:
C'est mieux de garder sign-in pour être iso ?

Reply:
Je vais demander à Emmy, merci

Reply:
On remplace par Log in With pour fwb et Pôle Emploi